akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/adapter/AdapterSpec.scala

```diff
@@ -157,6 +157,37 @@ class AdapterSpec extends AkkaSpec {
 
       typed1 should be theSameInstanceAs typed2
     }
+
+    "not crash if guardian is stopped" in {
+      for { _ ← 0 to 10 } {
+        var system: akka.actor.typed.ActorSystem[NotUsed] = null
+        try {
+          system = ActorSystem.create(Behaviors.deferred[NotUsed](_ ⇒ Behavior.stopped[NotUsed]), "AdapterSpec-stopping-guardian")
+        } finally {
+          if (system != null) {
+            Await.result(system.terminate(), remainingOrDefault)
```

Comment:
I prefer that we use the `shutdown` method in testkit, since it gives better reporting if it fails, and we have the timeout in a single place

Reply:
I see, good to know.